src/g_game.c

```diff
@@ -3321,6 +3321,7 @@ void G_InitNew(skill_t skill, int episode, int map)
   paused = false;
   demoplayback = false;
   automapactive = false;
+  automapfirststart = true;
```

Comment:
DO you think you could move this to `AM_LevelInit()`?

Reply:
Just tested, it seems to work and I'm convinced that it is a better approach. In a minute.